.github/workflows/deploy.yaml

```diff
@@ -17,18 +17,25 @@ jobs:
           node-version: 14
           registry-url: https://registry.npmjs.org/
 
-      - name: Restore cache of node modules
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+
+      - name: Restore cache of node modules and yarn cache
         uses: actions/cache@v2
         with:
           path: |
             node_modules
             */*/node_modules
-          key: ${{ runner.os }}-lerna-${{ hashFiles('**/yarn.lock') }}
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-nodejs-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-nodejs-
 
       - name: Install dependencies
         run: |
           yarn
-          node_modules/.bin/lerna bootstrap
+          yarn lerna bootstrap
```

Comment:
I don't think we have `lerna` in our `package.json`, do we? Would this work w/o it?

Reply:
lerna is a dependency in package.json

Reply:
It is, but it isn't listed in `package.json`, so it won't run.
![image](https://user-images.githubusercontent.com/1128312/117162915-79e1c800-adc3-11eb-90be-a4157bd4241c.png)
What we could do instead, is replace `yarn` for [`npx`](https://nodejs.dev/learn/the-npx-nodejs-package-runner), which because `lerna` is installed locally, it will effectively run `lerna bootstrap`

Reply:
^ scrap this, TIL that `yarn lerna bootstrap` _will_ actually look for `lerna` in our `node_modules`. It wasn't working in my machine because I didn't let `yarn` finish. Sorry!